### PR TITLE
bump(main/blink): 1.0.0

### PR DESCRIPTION
**Pull request by truboxl**

### CODE REVIEW

**Comment by truboxl** [comment]
This PR brings the cross compiled Blink on parity with on device building Blink.

The Blink VFS feature still need more work as it can potentially replace Proot+Blink or Proot+QEMU setup.
Blink VFS or Blink still lack the ability to fake root so Proot is needed. So not adding a separate blink-vfs package for now.

Some stuff still broken like unimplemented system calls (not sure Blink or Proot should handle this) which cause simple x86_64 Linux apt update and apt upgrade not working. A simple x86_64 Linux curl download works so YMMV.

TLDR Proot+Blink now works on some commands while others need to fallback to Proot+QEMU.

Will merge by end of today if no more reviews.
